src/visualization/include/iDynTree/Visualizer.h

```diff
@@ -466,6 +576,41 @@ class IModelVisualization
     virtual Transform getWorldLinkTransform(const LinkIndex& linkIndex) = 0;
 };
 
+class ITexture
+{
+public:
+
+    /**
+     * Destructor
+     */
+    virtual ~ITexture() = 0;
+
+    /**
+     * Return an interface to manipulate the texture environment.
+     */
+    virtual IEnvironment& environment() = 0;
+
+    /**
+     * @brief Get the color of the pixel at the given position in the additional texture.
+     *
+     * Remember to call draw() first.
+     * @param width The width of the pixel
+     * @param height The height of the pixel
+     * @return The color of the pixel
+     */
+    virtual ColorViz getPixelColor(unsigned int width, unsigned int height) const = 0;
+
+    /**
+     * @brief Get the pixels of the texture.
+     *
+     * Remember to call draw() first.
+     * @param pixels The output pixels
+     * @return True in case of success, false otherwise
+     */
```

Comment:
Can you clarify here: 
* the size of std::vector<PixelViz>
* if the pixel are stored in row major or col major order?

Now that this docs seems to be duplicated also in the concrete Texture implementation, so either update also that one, or remove it.

Reply:
I have updated the docs in https://github.com/robotology/idyntree/pull/802/commits/b7267fc3ba69363f2fcadd9fc53f37e1dd845d7f and removed the duplicated documentation in https://github.com/robotology/idyntree/pull/802/commits/d337204fa745b9b28cac986727a5f3d7bc5c9fd8.